pom.xml

```diff
@@ -156,46 +156,14 @@
                   <groupId>org.jacoco</groupId>
                   <artifactId>org.jacoco.agent</artifactId>
                   <version>${version.jacoco.plugin}</version>
+                  <classifier>runtime</classifier>
```

Comment:
What is the difference between regular agent JAR and `-runtime` one? Why didn't you add the `runtime` classifier to drools repo as well? Can we move the Surefire plugin configuration up to the parent too?

Reply:
ad the runtime JAR:
the documentation suggests this is the agent, while the regular one is "API to get a local copy of the agent" https://www.jacoco.org/jacoco/trunk/doc/repo.html. If you download it, the regular jar contains the agent jar inside plus some wrapper class. I added the classifier to make it consistent with the agent line, when the runtime jar is referred.

ad drools repo: good catch. As I already put the change in this PR (although it's not directly connected to moving the reporting to KIE parent), I will do it for drools too.

ad the Surefire plugin:
unfortunately, not every repository in KIE group can use the same configuration. Some (primarily engines) rely upon the runtime agent jar, while others (wb-related, containing GWT code) require using Jacoco maven plugin. For these two cases the Surefire plugin's configuration is different.

Reply:
The classifier has been added into the PR to Drools .